website/static/js/accountSettings.js

```diff
@@ -217,13 +217,13 @@ var UserProfileViewModel = oop.extend(ChangeMessageMixin, {
                     var addrText = $osf.htmlEscape(email.address());
                     bootbox.alert({
                                 title: 'Confirmation email sent',
-                                message: '<em>' + addrText + '</em>' + ' was added to your account.' +
-                                ' You will receive a confirmation email at ' + '<em>' + addrText + '</em>.' +
-                                ' Please log out of this account and check your email to confirm this action.',
+                                message: '<b>' + addrText + '</b>' + ' was added to your account.' +
+                                ' You will receive a confirmation email at ' + '<b>' + addrText + '</b>.' +
+                                ' Please check your email to confirm this action.',
                                 buttons: {
                                     ok: {
                                         label: 'Close',
-                                        className: 'btn-default'
+                                        className: 'btn-primary'
```

Comment:
Why was this changed? Other "Close" buttons seem to use `btn-default`: https://github.com/CenterForOpenScience/osf.io/blob/4f618ad25b8fbca86213bea604e708450021b1fd/website/static/js/pages/project-settings-page.js#L115-L118